service-api/pom.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>greencity</artifactId>
+        <groupId>Lv-427.Java</groupId>
+        <version>0.0.1-SNAPSHOT</version>
+    </parent>
+
+    <groupId>greencity</groupId>
+    <artifactId>service-api</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <name>service-api</name>
+
+    <dependencies>
+        <dependency>
```

Comment:
Lets remove those two spring dependencies for the moment.

Reply:
Ok.